runtime/server/api.go

```diff
@@ -83,3 +85,16 @@ func (s *Server) APIForName(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 }
+
+func errorResponse(err error) []byte {
```

Comment:
Use this new util instead – search for `httputil.Handler` in the code for example usage: https://github.com/rilldata/rill/blob/c752b64e89640b0299b21c27af3332df66a75bf6/runtime/pkg/httputil/httputil.go#L12